packages/rehype-mathjax/lib/output-svg.js

```diff
@@ -3,5 +3,11 @@ const Svg = require('mathjax-full/js/output/svg').SVG
 module.exports = createOutput
 
 function createOutput(options) {
-  return new Svg(options)
+  let settings = options || {}
+  if ('tex' in settings) {
+    settings = {...options}
+    delete settings.tex
+  }
+
+  return new Svg(settings)
```

Comment:
Can these changes in `output-svg` and `output-chtml` be done higher: can `lib/core.js` make parse and compile options instead?

And: does `mathjax` support unknown keys when they‘re set to undefined?

E.g. I’m thinking:

```js
var inputOptions = settings.tex
var outputOptions = Object.assign({}, settings)
// either of these two lines:
outputOptions.tex = undefined
delete outputOptions.tex
```

Reply:
MathJax would still throw with `outputOptions.tex = undefined`.
Moving the options format adjustment to the attacher required another flag (`browser`) to the `createPlugin` function because the options for the browser plugin correspond to the TeX input options, and not output options as in the non-browser plugins.